### DISCUSSION TITLE
[Docs] Going to production, ssr-caching

### DISCUSSION THREAD

**Discussion by awareness481** [vercel/next.js#27293] [Ideas]
Hey, I sincerely love the new [Going to production](https://nextjs.org/docs/going-to-production) docs. There's one issue though, the `ssr-caching` is referenced but it is broken in the last couple releases.

Should [this](https://github.com/vercel/next.js/issues/25621#issuecomment-855151444) band-aid fix be applied so users don't run into an error when using the example?

**Comment by awareness481** [comment]
cc: @leerob 

(thought it is appropriate to tag you, let me know If I shouldn't have :P)

**Reply by leerob** [reply]
The SSR caching example has been updated to show using cache-control headers. We recommend that over using a custom server, as you're deoptimizing your application when you eject. You only need a custom server if you need to change Next.js internal routing, usually.

https://github.com/vercel/next.js/tree/canary/examples/ssr-caching

**Reply by awareness481** [reply]
Oops, never figured that the example might have been updated as well :P
